bin/pap/pap.c

```diff
@@ -163,6 +166,8 @@ int main( int ac, char	**av)
     extern char		*optarg;
     extern int		optind;
 
+	int		is_imagewriter;
```

Comment:
Nitpick, but indentation should be four spaces and not a tab here.

In fairness, the indentation can be a bit all over the place in this codebase. But let's use the same indentation as neighboring code to maintain readability.